### PR TITLE
#3668 improved version check

### LINKED ISSUE #3668: Daily version check was disabled
With a new Keptn release, users get a notification of the new release in the Keptn Bridge. For some users, this was not working because they accidentally disabled the feature. 

**Success criteria:** Provide guidance for not disabling this feature. 

image

**Tasks:**
- Change the text: `As of now you can enable automatic version check to be notified about new version and security patches for Keptn and Keptn Bridge. This setting can be changed in the user menu on the top right.` > to > `Keptn provides an automatic version check to be notified about new versions and security patches for Keptn. It is recommended to have this check enabled. This setting can be changed in the user menu on the top right.`
- Change the text: `By enabling this feature, Keptn will collect statistical data and notify about new version and security patches for Keptn and Keptn Bridge.` > to > `By enabling this feature, Keptn will notify about new versions and security patches for Keptn  and therefore collect statistical data.`
- Add ` (recommended)` to the **Accept** button > [ Accept (recommended) ]
- Change the order of the buttons: [ Accept (recommended) ] [ Decline] 
- By default, set the feature flag for *Version check* to `enabled` (see screenshot above) - this is the default behavior as mentioned here: https://keptn.sh/docs/0.8.x/reference/version_check/
- Is there a cookie that stores the decision (Declined/Accepted)? Set the expiry date of the cookie for a Declined decision to 5 days.

**Definition of Done:**
- [ ] Documentation (screenshots) of the feature is updated: https://keptn.sh/docs/0.8.x/reference/version_check/
- [ ] As a user, I get more guidance to not disable the version check feature

### PR DESCRIPTION
**Pull request by Kirdock**
Closes #3668 
Related PR: https://github.com/keptn/keptn.github.io/pull/791
Signed-off-by: Klaus Strießnig <k.striessnig@gmail.com>

### CODE REVIEW

File: bridge/client/app/app-header/app-header.component.html [hunk 1]
```diff
@@ -59,19 +59,19 @@ <h3 class="m-0">Get started</h3>
         <p class="m-0">Version check</p>
       </div>
       <div>
-        <dt-switch (change)="versionCheckClicked($event)" [checked]="keptnInfo?.versionCheckEnabled"></dt-switch>
+        <dt-switch (change)="versionCheckClicked($event)" [checked]="keptnInfo?.versionCheckEnabled !== false"></dt-switch>
       </div>
     </div>
-    <p class="m-0 mt-2 small">By enabling this feature, Keptn will collect statistical data and notify about new version and security patches for Keptn and Keptn Bridge. Details can be found at <a [href]="versionCheckReference | keptnUrl" target="_blank" [textContent]="versionCheckReference | keptnUrl"></a></p>
+    <p class="m-0 mt-2 small">By enabling this feature, Keptn will notify about new versions and security patches for Keptn and therefore collect statistical data. Details can be found at <a [href]="versionCheckReference | keptnUrl" target="_blank" [textContent]="versionCheckReference | keptnUrl"></a></p>
```
Comment (jetzlstorfer) [inline bridge/client/app/app-header/app-header.component.html:65]: can we add "recommended" to this?

"By enabling this feature (recommended), Keptn will notify about new versions and security patches for Keptn and therefore collect statistical data."

File: bridge/client/app/app-header/app-header.component.html [hunk 2]
```diff
@@ -59,19 +59,19 @@ <h3 class="m-0">Get started</h3>
...
   </div>
 </dt-context-dialog>
 
 <dt-confirmation-dialog [state]="versionCheckDialogState" aria-label="Version check feature info">
   <dt-confirmation-dialog-state name="info">
-    As of now you can enable automatic version check to be notified about new version and security patches for Keptn and Keptn Bridge. This setting can be changed in the user menu on the top right.
+    Keptn provides an automatic version check to be notified about new versions and security patches for Keptn. It is recommended to have this check enabled. This setting can be changed in the user menu on the top right.
```
[the remark this answers is not in the document]
Comment (jsoref) [reply bridge/client/app/app-header/app-header.component.html:71]:
```suggestion
    By enabling this feature (recommended), Keptn will report when new versions and security patches for Keptn are available -- this involves collecting statistical data. This setting can be changed in the user menu on the top right.
```

[the remark this answers is not in the document]
Comment (jetzlstorfer) [reply bridge/client/app/app-header/app-header.component.html:71]: All the options are fine in my opinion. Please go ahead with the one that fits best with the rest of the text

**Review by ermin-muratovic — APPROVED** [review]
lgtm

**Pull request merged**